packages/haul-babel-preset-react-native/src/transforms/__tests__/stripDeadPlatformSelect.test.ts

```diff
@@ -0,0 +1,134 @@
+import stripDeadPlatformSelect from '../stripDeadPlatformSelect';
+import { transformSync } from '@babel/core';
+
+const fixture1 = `
+const value = Platform.select({
+  android: 'android',
+  ios: () => {},
+  default: { test: true },
+  web: 'what?'
+});
+`;
+
+const fixture2 = `
+const defaults = { default: 'default' };
+const value = Platform.select({
+  ios: () => {},
+  ...defaults,
+});
+`;
+
+const fixture3 = `
+const value = Platform.select({
+  android: () => {},
+});
+`;
+
+const fixture4 = `
+const value = Platform.select({
+  ios: false,
+  ['andr' + 'oid']: () => {},
+});
+`;
+
+const fixture5 = `
+const value = Platform.select({
+  default: 'default',
+  ['andr' + 'oid']() {}
+});
+`;
+
+const fixture6 = `
+const value = Platform.select({
+  ios: false,
+  android() {}
+});
+`;
+
+describe('stripDeadPlatformSelect transform', () => {
+  it('should replace Platform.select with value for default case if there is no matching platform', () => {
+    const { code } = transformSync(fixture1, {
```

Comment:
also grouping fixtures is harder for readability, can you move them inside the test cases?